challenge_3/style.css

```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.box {
+  background: #62306D;
```

Comment:
Let's use css custom properties (variables) for the colors.  See this example of its usage https://developer.mozilla.org/en-US/docs/Web/CSS/--*#css